docs/source/articles/payment-tender-types.html.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Payment Tender Types
+excerpt: TODO
+created_at: 2019/11/05
+---
+
+Payment Tender Types
+======================================================================
+
+TODO: document introduction and additional sections
+
+_Explain_ concepts in this doc.
+So that devs working through a specific tender type implementation doc (credit card, primary, advance payment) can make sense of the steps.
+Provide context.
+
+
+Gateway
+----------------------------------------------------------------------
+
+Each tender type relies on a gateway.
```

Comment:
Store credit doesn't, iirc

Reply:
I'm defining _gateway_ as "the object responsible for processing payments of that type". So for store credit, the gateway is `Payment::Profile`. That's the only definition I can come up with that holds up across the major tender types we've implemented in base and in plugins.